packages/expect/src/__tests__/__snapshots__/matchers.test.js.snap

```diff
@@ -4314,13 +4314,27 @@ exports[`toMatchObject() {pass: true} expect({"a": undefined}).toMatchObject({"a
 Expected: not <g>{"a": undefined}</>
 `;
 
+exports[`toMatchObject() {pass: true} expect({}).toMatchObject({"a": undefined, "b": "b", "c": "c"}) 1`] = `
+<d>expect(</><r>received</><d>).</>not<d>.</>toMatchObject<d>(</><g>expected</><d>)</>
+
+Expected: not <g>{"a": undefined, "b": "b", "c": "c"}</>
+Received:     <r>{}</>
```

Comment:
**question:** You'll notice that we don't include these non-enumerable keys in the error reporting. I have some ideas on how we can improve this, but fixing this will be quite involved. Is this acceptable for now? Can we handle improving this in a follow-up? 🤔

Reply:
a follow-up on this would be great, but we definitely don't have to handle it at the same time 👍